docs/getting-started.md

```diff
@@ -27,9 +27,9 @@ Semgrep requires Python 3.7 or later.
         ```sh
         docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep --config=auto
         ```
-2. Confirm installation by printing help manual page to your terminal. To do so, run the following command:
+2. Confirm installation by running a simple search pattern in the current directory. To do so, run the following command:
     ```sh
-    semgrep --help
+    semgrep --pattern '1' --lang generic
```

Comment:
```suggestion
    semgrep --pattern '127.$A.$B.$C' --lang generic /etc/hosts
```

This command might be more reliable. If a user ran the original query in their home directory, semgrep would keep scanning for minutes and yield hundreds of thousands of results.